addons/delivery/migrations/9.0.1.0/post-migration.py

```diff
@@ -26,6 +26,26 @@ def rename_property(cr, model, old_name, new_name):
         (new_name, field_ids))
 
 
+def reassign_carrier_id(cr):
+    cr.execute("""
+        select id, openupgrade_legacy_9_0_carrier_id from delivery_carrier
```

Comment:
It's better to use `openupgrade.get_legacy_name('carrier_id')`

Reply:
and also in line 32